prow/prstatus/prstatus.go

```diff
@@ -223,12 +223,11 @@ func (da *DashboardAgent) HandlePrStatus(queryHandler PullRequestQueryHandler) h
 				return
 			}
 
-			getSecret := func() []byte {
-				return []byte(token.AccessToken)
-			}
-
+			secretAgent := config.NewSecretAgent([]string{"AccessToken"})
+			secretAgent.SetSecret("AccessToken", []byte(token.AccessToken))
+			getToken := func() []byte { return []byte(token.AccessToken) }
 			// Construct query
-			ghc := github.NewClient(getSecret, githubEndpoint)
+			ghc := github.NewClient(getToken, config.NewReloadingTokenSource(getToken), githubEndpoint)
```

Comment:
nit: Feels weird to be able to create a `ReloadingTokenSouce` that does no reloading -- maybe just call the thing a more generic `TokenSource` since it does not enforce any reload ever happened?